tests/Unit/Evolution/EventsAndDenseTriggers/DenseTriggers/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY_SOURCES
```

Comment:
Does it satisfy CI to just put the sources in `tests/Unit/Evolution/EventsAndDenseTriggers/CMakeLists.txt`? I agree it's not elegant, but maybe this is a cmake versioning problem, and it would seem to work around the problem for now.

Reply:
You should be able to reproduce this warning by adding the `--warn-uninitialized` to `cmake`.

Reply:
I can, but the rest of the result is unexpected.
```
  uninitialized variable 'LAPACK_LINKER_FLAGS'
  uninitialized variable 'CPPCHECK_POSSIBLEERROR_ARG'
  uninitialized variable 'CPPCHECK_UNUSEDFUNC_ARG'
  uninitialized variable 'CPPCHECK_STYLE_ARG'
  uninitialized variable 'CPPCHECK_QUIET_ARG'
  uninitialized variable 'CPPCHECK_INCLUDEPATH_ARG'
  uninitialized variable 'LIBRARY_SOURCES'
  uninitialized variable 'BUILD_SHARED_LIBS'
```